### PR TITLE
Generation CLI Command Docs Only Refer to Charm Config

### PR DESCRIPTION
**Pull request by manadart**
## Description of change

Model generations will see a release targeting charm config, before later covering charm URL and resources. Until fully featured, help docs only refer to the charm config aspect.

## QA steps

None. No functional changes.

## Documentation changes

Possibly something for @pmatulis to be aware of.

## Bug reference

N/A

### CODE REVIEW

**Comment by manadart** [comment]
[Remove CLI Command Doc Refs to Charm URL and Resources](https://trello.com/c/mQrH02Dc/1260-remove-cli-command-doc-refs-to-charm-url-and-resources)

File: cmd/juju/model/checkout.go [hunk 1]
```diff
@@ -20,10 +20,9 @@ import (
 const (
 	checkoutSummary = "Work on the supplied branch."
 	checkoutDoc     = `
-Switch to the supplied branch, causing changes to charm configuration,
-charm URL or resources to apply only to units tracking the branch.
-Changing the branch to "master" causes changes to be applied to all units
-as usual.
+Switch to the supplied branch, causing changes to charm configuration to apply 
+only to units tracking the branch. Changing the branch to "master" causes 
+changes to be applied to all units as usual.
```
Comment (SimonRichardson) [inline cmd/juju/model/checkout.go]: I don't quite follow the last sentence, does this mean if you have changes on a unit and switching to "master" branch will cause them to be staged and committed?

Comment (manadart) [reply cmd/juju/model/checkout.go]: I've clarified to better represent master branch behaviour.

**Comment by manadart** [comment]
$$merge$$

**Comment by manadart** [comment]
$$merge$$

**Comment by manadart** [comment]
Now: `Could not resolve host: github.com`

$$merge$$

**Pull request merged**